spyder/api/plugins.py

```diff
@@ -12,6 +12,9 @@
 main window and call other plugins directly.
 """
 
+# Standard library imports
+import sys
```

Comment:
This is probably a leftover of a merge. Please remove it.

Reply:
@ccordoba12 I think im going to reverse the last 3 commits and do the merge again carefully so we don't have any problems.

Reply:
Ok, thanks! Do you need help with this? I can give you a hand, if you want.